client/sites/staging-site/components/staging-site-card/index.js

```diff
@@ -61,7 +61,7 @@ export const StagingSiteCard = ( {
 	const [ syncError, setSyncError ] = useState( null );
 	// eslint-disable-next-line no-unused-vars
 	const [ _, setIsErrorValidQuota ] = useState( false );
-	const [ progress, setProgress ] = useState( 0.1 );
+	const [ progress, setProgress ] = useState( 10 );
```

Comment:
`ProgressBar` component is quite limited (https://wordpress.github.io/gutenberg/?path=/docs/components-progressbar--docs&args=value:80) and does not offer a `max` property that would set the maximum value for the progress. It only assumes that the maximum value is 100 (%).

Because of that, we need to set the progress to 10-times what it is currently.